### PR TITLE
Update upload.mdx

### PR DESCRIPTION
**Pull request by murnifine**
#### Description

- What does this PR change? Give us a brief description. 
- Closes #

### CODE REVIEW

File: src/content/docs/plugin/upload.mdx [hunk 1]
```diff
@@ -80,7 +80,7 @@ import { upload } from '@tauri-apps/plugin-upload';
 upload(
   'https://example.com/file-upload',
   './path/to/my/file.txt',
-  (progress, total) => console.log(`Uploaded ${progress} of ${total} bytes`), // a callback that will be called with the upload progress
+  (uploading) => console.log(`Uploaded ${uploading.progress} of ${uploading.total} bytes`), // a callback that will be called with the upload progress
```
Comment (FabianLars) [inline src/content/docs/plugin/upload.mdx]: Can we make this the same as below? So like this:
```suggestion
  ({ progress, total }) => console.log(`Uploaded ${progress} of ${total} bytes`), // a callback that will be called with the upload progress
```

Comment (murnifine) [reply src/content/docs/plugin/upload.mdx]: like that can also be done

File: src/content/docs/plugin/upload.mdx [hunk 2]
```diff
@@ -93,7 +93,7 @@ import { download } from '@tauri-apps/plugin-upload';
 download(
   'https://example.com/file-download-link',
   './path/to/save/my/file.txt',
-  (progress, total) => console.log(`Downloaded ${progress} of ${total} bytes`), // a callback that will be called with the download progress
+  ({progress, total}) => console.log(`Downloaded ${progress} of ${total} bytes`), // a callback that will be called with the download progress
```
Comment (FabianLars) [inline src/content/docs/plugin/upload.mdx]:
```suggestion
  ({ progress, total }) => console.log(`Downloaded ${progress} of ${total} bytes`), // a callback that will be called with the download progress
```

**Review by FabianLars — APPROVED** [review]
thank you :)

**Pull request merged**